SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/SecureVaultStorage/DataBrokerProtectionDatabase.swift

```diff
@@ -634,7 +634,7 @@ extension DataBrokerProtectionDatabase {
 
         let newProfileQueries = profile.profileQueries
 
-        let databaseBrokerProfileQueryData = try fetchAllBrokerProfileQueryData()
+        let databaseBrokerProfileQueryData = try fetchAllBrokerProfileQueryData(shouldFilterRemovedBrokers: false)
```

Comment:
I wonder if this should be true since we won't want to run operations for removed brokers

Reply:
That’s a good point, I was unsure if we would want to update profiles. Technically, it will update again if the broker ends up being de-removed, so no need to keep it up to date while the broker is removed.